src/integrationTest/java/uk/gov/hmcts/reform/ccd/documentam/controller/CaseDocumentAmControllerIT.java

```diff
@@ -603,28 +598,28 @@ private StoredDocumentHalResource getStoredDocumentResourceToUpdatePatch(Date ti
                 "GUARDED",
                 CASE_TYPE_ID_VALUE,
                 JURISDICTION_ID_VALUE,
-                "The Security Classification is not valid"
+                "Classification is not valid"
```

Comment:
Could this not use value from `Constants.java` ?

Reply:
I think this should remain as is.  This way ensures that we are actually testing the values in `Constants.java`.